test/cql-pytest/test_cast.py

```diff
@@ -49,7 +49,7 @@ def table2(cql, test_keyspace):
 # It's important that it stays this way - in the future we might implement guessing the type of untyped constants
 # by assigning the smallest type that can fit the constant, but I think that we shouldn't allow converting them
 # to blob without explicitly specifying size of the integer.
-def test_cast_int_literal_to_blob(cql, table1):
+def test_cast_int_literal_to_blob(cql, table1, scylla_only):
```

Comment:
I think you added scylla_only to this test by mistake?

Reply:
Right, it's not needed here, fixed.